python-clusters/attach-eks-cluster/cluster.py

```diff
@@ -68,6 +68,9 @@ def start(self):
         with open(kube_config_path, 'w') as f:
             f.write(kube_config_str)
 
+        logging.info("Cluster attached. Cluster configuration:")
```

Comment:
nit:
```suggestion
        logging.info("Cluster attached. Cluster configuration (kube_config):")
```

Reply:
> Is it worth a line in the changelog ? (I don't think so, but just raising the point)

I don't think it is worth it except if it is the only change in the plugin (as there would be no entry in the changelog for this particular version)